lib/CL/devices/hsa/pocl-hsa.c

```diff
@@ -118,6 +126,112 @@ pocl_hsa_get_agents(hsa_agent_t agent, void *data)
   return HSA_STATUS_SUCCESS;
 }
 
+//Get hsa-unsupported device features from hsa device list.
+int num_hsa_device = 1;
+hsa_device_features hsa_device_lists[MAX_HSA_AGENTS];
```

Comment:
Can you rename this to "hsa\_supported\_device\_properties" or similar and make it 'static'. Also, is init\_hsa\_device\_list() necessary, cannot you use a compile-time array/struct initializer?